src/dispatch/plugins/dispatch_google/groups/plugin.py

```diff
@@ -78,6 +78,9 @@ def add_member(client: Any, group_key: str, email: str, role: str):
         except HttpError as e:
             #  we are okay with duplication errors upon insert
             if e.resp.status in [409]:
+                log.debug(
+                    f"Member already exists in google group. GroupKey={group_key} Body={body}"
```

Comment:
Can we use the error string we get from Google instead?

Reply:
We could, but dumping the json into the logs make it less readable imo.

Reply:
I guess that's fine as long as the response status 409 is only used for this type of error.

Reply:
It looks like it's pretty standard: https://developers.google.com/webmaster-tools/search-console-api-original/v3/errors#:~:text=CONFLICT%20(409),-Error%20code&text=The%20API%20request%20cannot%20be,identified%20with%20more%20specific%20errors.